### PR TITLE
Implement view for Order copy cards payment form

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-767

Implement view for order copy cards payment form
Screenshot 2019-12-09 at 22 51 20

### CODE REVIEW

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    <h1 class="heading-large"><%= t(".heading") %></h1>
+
+    <div class="form-group">
+      <table>
+        <tbody>
+          <tr>
+            <th>
+              <span class="strong">
+                <% if @transient_registration.temp_cards > 1 %>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]: Did you know you can do this:

```
t(".total_cost", count: @transient_registration.temp_cards)
```

And then in the YAML have:

```
cost:
  one: "1 card"
  other: "%{count} cards"
```

And it'll handle all the logic for you? It's pretty handy :)

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                  <%= t(
+                        ".total_cost.multiples",
+                        cards: @transient_registration.temp_cards,
+                        cost: display_pence_as_pounds(@transient_registration.total_registration_card_charge)
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]: It looks like you aren't doing anything with the `cost` argument.

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      </table>
+    </div>
+
+    <p><%= t(".vat_message") %></p>
+
+    <div class="form-group <%= "form-group-error" if @copy_cards_payment_form.errors[:temp_payment_method].any? %>">
+      <h2 class="heading-medium">
+        <%= t(".payment_method.subheading") %>
+      </h2>
+      <p class='form-hint'><%= t(".payment_method.legend") %></p>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]:
```suggestion
      <p class="form-hint"><%= t(".payment_method.legend") %></p>
```

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          <%= f.radio_button :temp_payment_method, "bank_transfer" %>
+          <%= f.label :temp_payment_method, value: "bank_transfer", class: "form-label" do %>
+            <%= t(".options.bank_transfer") %>
+            <span class='form-hint'><%= t(".hint_pay_by_bank_transfer") %></span>
+          <% end %>
+        </div>
+      </fieldset>
+    </div>
+
+    <%= f.hidden_field :reg_identifier, value: @copy_cards_payment_form.reg_identifier %>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]: We can ditch this now, right?

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            <%= t(".options.card") %>
+            <span class='form-hint'><%= t(".hint_pay_by_card") %></span>
+          <% end %>
+        </div>
+
+        <div class="multiple-choice">
+          <%= f.radio_button :temp_payment_method, "bank_transfer" %>
+          <%= f.label :temp_payment_method, value: "bank_transfer", class: "form-label" do %>
+            <%= t(".options.bank_transfer") %>
+            <span class='form-hint'><%= t(".hint_pay_by_bank_transfer") %></span>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]:
```suggestion
            <span class="form-hint"><%= t(".hint_pay_by_bank_transfer") %></span>
```

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      <fieldset id="temp_payment_method">
+        <% if @copy_cards_payment_form.errors[:temp_payment_method].any? %>
+          <span class="error-message"><%= @copy_cards_payment_form.errors[:temp_payment_method].join(", ") %></span>
+        <% end %>
+
+        <div class="multiple-choice">
+          <%= f.radio_button :temp_payment_method, "card" %>
+          <%= f.label :temp_payment_method, value: "card", class: "form-label" do %>
+            <%= t(".options.card") %>
+            <span class='form-hint'><%= t(".hint_pay_by_card") %></span>
```
Comment (irisfaraway) [inline app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]:
```suggestion
            <span class="form-hint"><%= t(".hint_pay_by_card") %></span>
```

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb (hunk 1 above)
Comment (cintamani) [reply app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]: 😮 I didn't! TIL! Thank :) will make the change!

File: app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb (hunk 2 above)
Comment (cintamani) [reply app/views/waste_carriers_engine/copy_cards_payment_forms/new.html.erb]: Oooohhh yeah! good catch!

**Pull request merged**